gulp-tasks/build-packages.js

```diff
@@ -6,17 +6,33 @@
   https://opensource.org/licenses/MIT.
 */
 
-const fse = require('fs-extra');
+const del = require('del');
+const fs = require('fs-extra');
 const gulp = require('gulp');
 const path = require('path');
 
 const constants = require('./utils/constants');
 const packageRunnner = require('./utils/package-runner');
 
-const cleanPackage = (packagePath) => {
-  const outputDirectory = path.join(packagePath,
-      constants.PACKAGE_BUILD_DIRNAME);
-  return fse.remove(outputDirectory);
+const cleanPackage = async (packagePath) => {
+  // Delete generate files from the the TypeScript transpile.
+  if (await fs.pathExists(path.join(packagePath, 'src', 'index.ts'))) {
+    const deletedPaths = await del([
+      path.join(packagePath, '**/*.+(mjs|d.ts)'),
```

Comment:
You probably want `path.posix.join()` here, since I believe these patterns should always use `/`, event when run on Windows.

Reply:
I *think* you might be right. I know that's true for `node-glob`, but I believe `del` uses `globby`, so it may be different?